packages/core/admin/admin/src/StrapiApp.js

```diff
@@ -71,14 +102,14 @@ class StrapiApp {
       const boot = this.appPlugins[plugin].boot;
 
       if (boot) {
-        boot(this);
+        boot({ getPlugin: this.getPlugin });
       }
     });
   }
 
-  getPlugin(pluginId) {
+  getPlugin = pluginId => {
     return this.plugins[pluginId] || null;
-  }
+  };
```

Comment:
`return this.plugins[pluginId]` will yield undefined anyways if it does not exist, you can drop the || null :)